src/commands/deploy/legacy.ts

```diff
@@ -1255,12 +1297,12 @@ function handleCreateDeployError(output: Output, error: Error) {
       return 1;
     }
 
-    const link = 'https://zeit.co/docs/v2/deployments/configuration/';
-
     output.error(
       `Failed to validate ${highlight(
         'now.json'
-      )}: ${message}\nDocumentation: ${link}`
+      )}: ${message}\nDocumentation: ${
+        link('https://zeit.co/docs/v2/deployments/configuration/')
```

Comment:
```suggestion
        link('https://zeit.co/docs/v2/advanced/configuration')
```

This url has changed